fusequery/query/src/servers/mysql/mysql_session.rs

```diff
@@ -72,13 +73,19 @@ impl AbortableService<TcpStream, ()> for Session {
                 "Cannot to convert Tokio TcpStream to Std TcpStream"
             })?;
 
-        let session_id = self.get_id();
         let cloned_stream = stream.try_clone()?;
         let session = session_manager.get_session(&self.session_id)?;
 
         std::thread::spawn(move || {
             session.get_status().lock().enter_init(cloned_stream);
 
+            let session_ref = session.clone();
+            exit_scope!({
```

Comment:
Very hardcore, mysql  client works now.
clickhouse handler need this change too? Now it stucks after the panic.

Reply:
ClickHouse handler doesn't support graceful shutdown yet.

Reply:
https://github.com/datafuselabs/datafuse/issues/832

Reply:
Oh I see.
Thanks